src/api/db/data/20200326221616_regenerate_notifications.rb

```diff
@@ -91,8 +91,11 @@ def recent_request_comments
 
   def create_notifications(event)
     event.subscriptions.each do |subscription|
-      rss_params = Notification::RssFeedItem.rss_notification_params(event, subscription)
-      Notification::RssFeedItem.create(rss_params)
+      next if subscription.subscriber && subscription.subscriber.away?
+      params = subscription.parameters_for_notification.merge!(event.parameters_for_notification)
+      Notification::RssFeedItem.create(params)
+    rescue StandardError => e
+      Airbrake.notify(e, event_id: event.id)
```

Comment:
What's the difference to "just" let the exception populate to errbit?

Reply:
Let's talk about this.

Reply:
We talked and came to the conclusion to leave this alone for now